### PR TITLE
add known issue for operator

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

**Comment by rojkov** [comment]
reference #476

File: cmd/operator/README.md [hunk 1]
```diff
@@ -78,4 +79,13 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
...
+The samples for them are available [here](/deployments/operator/samples/).
+
+## Known issues
+
+When the operator is run with leader election enabled, that is with the option
+`--enable-leader-election`, make sure the cluster is not overloaded with excessive
+number of pods. Otherwise a heart beat used by the leader election code may trigger
+a timeout and crash. We are going to use different clients for the controller and
+leader election code to alleviate the issue, but in case of an overloaded cluster
+the controller's client will crash anyway when it hits a timeout.
```
Comment (bart0sh) [inline cmd/operator/README.md:91]: This doesn't sound too optimistic to me. Should we give a reader at least some hope for a possible future fix?

Comment (rojkov) [reply cmd/operator/README.md:91]: We can simply drop the last "but" to make it sound less morbid :)

Comment (bart0sh) [reply cmd/operator/README.md:91]: sgtm

Comment (rojkov) [reply cmd/operator/README.md]: done

**Review by bart0sh — APPROVED** [review]

File: cmd/operator/README.md [hunk 2]
```diff
@@ -78,4 +79,12 @@ $ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/
...
\ No newline at end of file
+The samples for them are available [here](/deployments/operator/samples/).
+
+## Known issues
+
+When the operator is run with leader election enabled, that is with the option
+`--enable-leader-election`, make sure the cluster is not overloaded with excessive
+number of pods. Otherwise a heart beat used by the leader election code may trigger
+a timeout and crash. We are going to use different clients for the controller and
+leader election code to alleviate the issue.
```
Comment (mythi) [inline cmd/operator/README.md:90]: Perhaps add a reference to #476 so that the reader is able to see the progress, like "See more details in #476"

Comment (rojkov) [reply cmd/operator/README.md]: Added link to #476

**Review by mythi — APPROVED** [review]
+1

**Review by bart0sh — APPROVED** [review]

**Pull request merged**